EDMarketConnector.py

```diff
@@ -925,56 +935,89 @@ def getandsend(self, event=None, retrying: bool = False):  # noqa: C901, CCR001
             self.button['state'] = self.theme_button['state'] = tk.DISABLED
             self.w.update_idletasks()
 
+        query_time = int(time())
+        logger.trace_if('capi.worker', 'Requesting full station data')
+        companion.session.station(
+            query_time=query_time, tk_response_event=self._CAPI_RESPONSE_TK_EVENT_NAME,
+            retrying=retrying, play_sound=play_sound
+        )
+        config.set('querytime', query_time)
```

Comment:
Yes, moved it up for now.  Ideally this would get set exactly where we (first) make an actual CAPI query, as the idea is to be on cooldown versus that.  But that will require more-intrusive changes.